Lib/concurrent/futures/process.py

```diff
@@ -477,6 +477,15 @@ def _terminate_broken(self, cause):
         if cause is not None:
             bpe.__cause__ = _RemoteTraceback(
                 f"\n'''\n{''.join(cause)}'''")
+        else:
+            # No cause known, so try to report some helpful info about
+            # which process(es) terminated and with what exit code
+            errors = []
+            for p in self.processes.values():
+                if p.exitcode:  # Report any nonzero exit codes
```

Comment:
Instead of a comment, let's just make it self-documenting:

```suggestion
                if p.exitcode != 0:
```

Reply:
This is slightly different - subprocesses that are not yet terminated will have `exitcode = None`, so just comparing with `0` will match `None` as well. That said, I think self-documenting is definitely the way to go, so I'll make the check more explicit: `if p.exitcode is not None and p.exitcode != 0:`